rockcraft.yaml

```diff
@@ -20,7 +20,7 @@ parts:
   certificates:
     plugin: nil
     stage-packages:
-      - ca-certificates
+      - ca-certificates_data
```

Comment:
Do we need the certificates in hydra? I guess there is no harm in having the there even if they are not used right now?